.github/workflows/pr-approved.yml

```diff
@@ -0,0 +1,68 @@
+name: PR approved
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  check-skip:
+    name: Check if integration tests should run
+    # NOTE: the name of the special label is hardcoded here
+    # it would be better to extract it to a more global location, e.g. the workflow-level env context,
+    # but the env context is not available in job-level if expressions (only step-level ones)
+    if: contains(github.event.label.name, 'approved')
```

Comment:
@lbianchi-lbl Quick question: do we  need to add the approved label manually or is this automatically added once two approving reviews are available?

Reply:
Yes, the `approved` label is managed automatically by the `pr-review.yml` workflow:

- Added when the second approving review is submitted
- Removed if the label is present and there are fewer than 2 approving reviews
- Left as-is in the remaining cases